### PR TITLE
NMS-14721: Quick Start: Review entire quick start section when complete

### PR DESCRIPTION
**Pull request by Bonrob2**
### All Contributors

* [ ] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-14721
* http://issues.opennms.org/browse/NMS-14984
* http://issues.opennms.org/browse/NMS-14985
* http://issues.opennms.org/browse/NMS-14999

### CODE REVIEW

**Review by emarsh1995 — CHANGES_REQUESTED** [review]
We're nearly there!

File: docs/modules/operation/pages/quick-start/notification-config.adoc [hunk 1]
```diff
@@ -64,3 +71,7 @@ Topics include:
 
 * xref:operation:deep-dive/notifications/concepts.adoc[Concepts (events, UEIs, on-call roles, and so on)]
 * xref:operation:deep-dive/notifications/commands.adoc[Notification commands]
+
+
+For example, defining a five minute delay will mean that, when a notification is generated, the first member of the group will be notified immediately.
+Subsequent members will then be notified in five-minute increments, from the top of the list of group members to the bottom.
```
[the remark this answers is not in the document]
Comment (Bonrob2) [reply docs/modules/operation/pages/quick-start/notification-config.adoc:77]: Hmm. No. Will update and remove. Thanks.

File: docs/modules/operation/pages/quick-start/notifications.adoc [hunk 2]
```diff
@@ -27,58 +27,71 @@ This configuration file update happens immediately, with no need to restart {pag
...
-.. (Optional) Define a delay interval for group notification.
-This will determine the delay between sending notifications to each member of a group.
-+
-For example, defining a five minute delay will mean that, when a notification is generated, the first member of the group will be notified immediately.
-Subsequent members will then be notified in five-minute increments, from the top of the list of group members to the bottom.
-.. Click *Next Step*.
-.. Select `javaEmail` from the list of commands and make sure it is on.
+.. You are prompted to define the time to wait between sending notifications to each member of a group.
+For the purpose of this Quick Start guide, do not specify a delay interval, and click *Next Step*.
+.. Select `browser` from the list of commands and make sure that it is on.
```
Comment (emarsh1995) [inline docs/modules/operation/pages/quick-start/notifications.adoc:91]:
```suggestion
.. Select `Browser` from the list of commands and make sure that it is on.
```

**Review by emarsh1995 — CHANGES_REQUESTED** [review]
Just a few minor nits. :)

File: docs/modules/operation/pages/deep-dive/notifications/concepts.adoc [hunk 3]
```diff
@@ -77,6 +77,9 @@ When a step targets one or more groups, you may specify a delay for each group.
 The default delay is zero seconds; this means all group members are notified at the same time.
```
Comment (emarsh1995) [inline docs/modules/operation/pages/deep-dive/notifications/concepts.adoc:77]:
```suggestion
The default delay is zero seconds; this means that all group members are notified at the same time.
```

File: docs/modules/operation/pages/deep-dive/notifications/concepts.adoc [hunk 4]
```diff
@@ -77,6 +77,9 @@ When a step targets one or more groups, you may specify a delay for each group.
...
 If you set a longer delay, group members will be notified in alphabetical order, based on their usernames.
 
+For example, defining a five minute delay will mean that when a notification is generated the first member of the group will be notified immediately.
```
Comment (emarsh1995) [inline docs/modules/operation/pages/deep-dive/notifications/concepts.adoc:80]:
```suggestion
For example, defining a five minute delay will mean that when a notification is generated, the first member of the group is notified immediately.
```

File: docs/modules/operation/pages/deep-dive/notifications/concepts.adoc [hunk 5]
```diff
@@ -77,6 +77,9 @@ When a step targets one or more groups, you may specify a delay for each group.
...
+Subsequent members will then be notified in five-minute increments, from the top of the list of group members to the bottom.
```
Comment (emarsh1995) [inline docs/modules/operation/pages/deep-dive/notifications/concepts.adoc:81]:
```suggestion
Subsequent members are then notified in five-minute increments, from the top of the list of group members to the bottom.
```

File: docs/modules/operation/pages/deep-dive/notifications/configuration.adoc [hunk 6]
```diff
@@ -0,0 +1,23 @@
+
+= Notification configuration
```
Comment (emarsh1995) [inline docs/modules/operation/pages/deep-dive/notifications/configuration.adoc:2]:
```suggestion
= Notification Configuration
```

File: docs/modules/operation/pages/deep-dive/notifications/configuration.adoc [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+The xref:operation:quick-start/notification-config.adoc[configure notifications] chapter in the Quick Start section includes basic procedures to configure and customize {page-component-title} notifications.
```
Comment (emarsh1995) [inline docs/modules/operation/pages/deep-dive/notifications/configuration.adoc:4]:
```suggestion
The xref:operation:quick-start/notification-config.adoc[Configure Notifications] chapter in the Quick Start section includes basic procedures to configure and customize {page-component-title} notifications.
```

File: docs/modules/operation/pages/deep-dive/notifications/configuration.adoc [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+[[path-outage-notification]]
+== Configure path outages
+
+Path outages prevent the generation of excess notifications by identifying network paths where failures may cause multiple nodes to appear offline.
+
+Configuring a path outage lets you suppress notifications for nodes that appear to be down due to a failure in the network path between the nodes and {page-component-title}.
+For example, if a WAN link fails, all nodes at the remote site served by the WAN link will appear to be down.
+Because you get a notification that the router on the far end of the WAN link is not responding, you don't need notifications for all devices that sit behind that router.
+
+See xref:operation:deep-dive/service-assurance/path-outages.adoc[use path outages to suppress notifications] for additional information.
```
Comment (emarsh1995) [inline docs/modules/operation/pages/deep-dive/notifications/configuration.adoc:16]:
```suggestion
See xref:operation:deep-dive/service-assurance/path-outages.adoc[Use Path Outages to Suppress Notifications] for additional information.
```

File: docs/modules/operation/pages/quick-start/notification-config.adoc [hunk 9]
```diff
@@ -44,23 +49,11 @@ That means if there is an event that is not associated with a node or interface
...
-. Click *Next* to save the configuration.
-
 == Beyond Quick Start
 
 For more information on notifications, refer to xref:operation:deep-dive/notifications/introduction.adoc[Notifications] in the Deep Dive section.
 Topics include:
 
-* xref:operation:deep-dive/notifications/concepts.adoc[Concepts (events, UEIs, on-call roles, and so on)]
+* xref:operation:deep-dive/notifications/concepts.adoc[Concepts] (events, UEIs, on-call roles, and so on)
+* xref:operation:deep-dive/notifications/configuration.adoc#path-outage-notification[Configure path outages]
```
Comment (emarsh1995) [inline docs/modules/operation/pages/quick-start/notification-config.adoc:58]:
```suggestion
* xref:operation:deep-dive/notifications/configuration.adoc#path-outage-notification[Path outages]
```

**Review by emarsh1995 — CHANGES_REQUESTED** [review]
A couple more changes, then I think we're good to go!

**Review by mhuot — APPROVED** [review]
Looks good to me, good work

File: docs/modules/deployment/pages/core/getting-started.adoc [hunk 10]
```diff
@@ -171,6 +171,35 @@ To achieve this, use the `scvcli` command line utility to add the credentials, a
                     password="${scv:postgres-admin:password}" />
 ----
 
+[[ga-data-choices]]
+== First-time login and data choices
+
+After you start the {page-component-title} core services, access the web application at `\http://core-instance-ip:8980/opennms`.
```
Comment (emarsh1995) [inline docs/modules/deployment/pages/core/getting-started.adoc:177]:
```suggestion
After you start the {page-component-title} core services, you can access the web application at `\http://core-instance-ip:8980/opennms`.
```

File: docs/modules/operation/pages/deep-dive/notifications/configuration.adoc [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+
+Configuring a path outage lets you suppress notifications for nodes that appear to be down due to a failure in the network path between the nodes and {page-component-title}.
+For example, if a WAN link fails, all nodes at the remote site served by the WAN link will appear to be down.
+Because you get a notification that the router on the far end of the WAN link is not responding, you don't need notifications for all devices that sit behind that router.
+
+See xref:operation:deep-dive/service-assurance/path-outages.adoc[Use Path Outages to Suppress Notifications] for additional information.
+
+You can configure a path outage for an individual node by setting a parent on the node in a requisition:
+
+. Click the *Gear* symbol in the top-right of the screen.
```
Comment (emarsh1995) [inline docs/modules/operation/pages/deep-dive/notifications/configuration.adoc:20]: Bringing this in line with our discussion about symbols. :)

```suggestion
. Click the *gear* symbol in the top-right of the screen.
```

File: docs/modules/operation/pages/quick-start/notification-config.adoc [hunk 12]
```diff
@@ -11,13 +11,18 @@ We assume that you have followed the previous steps and are ready to proceed to
...
+You can configure notifd to process only the destination path of the first matching notification by editing `$\{OPENNMS_HOME}/etc/notifd-configuration.xml` and setting `match-all="false"` in the top-level `notifd-configuration` element.
+
+NOTE: Changes to `$\{OPENNMS_HOME}/etc/destinationPaths.xml` happen immediately, with no need to restart {page-component-title}.
 
 == Configure event notifications
 
-An event notification defines a destination path and message template for a specific event UEI.
+An event notification defines a destination path and message template for a specific UEI.
 It is possible to define multiple notifications for the same UEI to use different filters, destination paths, and message templates.
 Follow these steps to configure event notifications:
```
Comment (emarsh1995) [inline docs/modules/operation/pages/quick-start/notification-config.adoc:27]: There's another instance of the gear symbol here, at line 29, that I can't get to.

File: docs/modules/operation/pages/quick-start/notifications.adoc [hunk 13]
```diff
@@ -1,11 +1,11 @@
...
 
-{page-component-title} uses notifications to inform users about events in the network without requiring them to log in to the web UI.
-Notifications are commonly sent through email and text/pager messages, but can be customized to send to a variety of external systems.
-For additional information, including alternative notification mechanisms, see xref:operation:deep-dive/notifications/introduction.adoc[Notifications] in the Deep Dive section and the xref:reference:daemons/daemon-config-files/notifd.adoc[notifd daemon] reference section.
+{page-component-title} uses notifications to inform users about xref:operation:deep-dive/events/introduction.adoc[events] in the network without requiring them to log in to the web UI.
+Notifications are commonly sent through email and text or pager messages, but can be customized to send to a variety of external systems.
+For additional information, including alternative notification mechanisms, see xref:operation:deep-dive/notifications/introduction.adoc[Notifications] in the Deep Dive section and the xref:reference:daemons/daemon-config-files/notifd.adoc[notifd] reference section.
 
 == Enable notifications
```
Comment (emarsh1995) [inline docs/modules/operation/pages/quick-start/notifications.adoc:10]: Another instance of the gear symbol at line 19.

File: docs/modules/operation/pages/quick-start/notifications.adoc [hunk 14]
```diff
@@ -27,58 +27,71 @@ This configuration file update happens immediately, with no need to restart {pag
...
 
 You must specify a destination path to receive notifications from {page-component-title}.
 The `Email-Admin` destination path is defined by default, and is the notification path when no other destination paths are specified.
 It sends email notifications to users in the Admin group.
 
-This example specifies a destination path that sends notifications to users in the Admin group (currently just the admin user) and to an email address that you specify.
+For the `Email-Admin` destination path to work, you have to configure your email server, which is beyond the scope of this Quick Start guide.
+For this example, we will specify a destination path that sends browser pop-up notifications to users in the Admin group (currently, the default admin user and the personalized user that you created in xref:operation:quick-start/users.adoc#create-user[Step 1: First-Time Login]).
 
 . Click the *Gear* symbol in the top-right of the screen.
```
Comment (emarsh1995) [inline docs/modules/operation/pages/quick-start/notifications.adoc:81]:
```suggestion
. Click the *gear* symbol in the top-right of the screen.
```

File: docs/modules/operation/pages/quick-start/users.adoc [hunk 15]
```diff
@@ -1,43 +1,19 @@
...
 
 [[create-user]]
 == Create a new user
 
+We recommend that you do not use the default `admin` user account, but instead xref:deep-dive/user-management/user-config.adoc[create specific users] with the Admin role and other permissions.
+This helps to keep track of who has performed tasks such as clearing alarms or creating notifications, set up personalized notification workflows or group users into teams to help delegate areas of responsibility within {page-component-title}.
+
 For the purpose of this Quick Start guide, create one new user account for yourself, assign it the Admin role, and configure your email address and any other relevant contact information:
 
 . Click the *Gear* symbol at the top-right of the screen.
```
Comment (emarsh1995) [inline docs/modules/operation/pages/quick-start/users.adoc:19]:
```suggestion
. Click the *gear* symbol at the top-right of the screen.
```

**Review by emarsh1995 — CHANGES_REQUESTED** [review]
A couple little nits, mostly just to update based on our discussion about referring to symbols in the UI. :)

**Pull request merged**